src/transport/raweth/rx.c

```diff
@@ -117,6 +117,26 @@ z_result_t _z_raweth_recv_t_msg(_z_transport_multicast_t *ztm, _z_transport_mess
     return _z_raweth_recv_t_msg_na(ztm, t_msg, addr);
 }
 
+z_result_t _z_raweth_update_rx_buff(_z_transport_multicast_t *ztm) {
+    // Check if user or defragment buffer took ownership of buffer
+    if (!_z_zbuf_is_last_ref(&ztm->_zbuf)) {
+        // Allocate a new buffer
+        _z_zbuf_t new_zbuf = _z_zbuf_make(Z_BATCH_MULTICAST_SIZE);
```

Comment:
I don't know the raweth very well, but should we use Z_BATCH_MULTICAST_SIZE there?

Reply:
Yes because raweth is a mix between a link and a multicast transport so it should behave as closely as possible as such.